### DISCUSSION TITLE
키글챌 주간 통계 - 10주차 (7월 24일 ~ 7월 30일)

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#131] [주간 통계]
### ✍️ 멤버별 작성한 글 수:
*: 피드백으로 대체

| 닉네임 | 게시물 수 |
| -------- | ---------- |
| 이다은 |   0   |
| 민예진 |   0   |
| 손현경 |   0   |
| 송세연 |   1   |
| 신채원 |   0   |
| 장서윤 |   0   |

### 챌린지를 수행하지 않은 멤버: 5명
이다은, 민예진, 손현경, 신채원, 장서윤

### 💪 8월 6일까지 반성문을 작성해 반성문 카테고리에 게시해주세요

**Comment by shkisme** [comment]
으아아아아아앙 😂
